### PR TITLE
added wget and highlight other mode

### PR DESCRIPTION
**Pull request by shoffman-percona**
added instructions for using wget (curl not part of ubuntu default install)
Added information about interactive mode for users that want to customize installation parameters from our defaults

### CODE REVIEW

**Comment by rnovikovP** [comment]
@shoffman-percona , @nikita-b  can  we have this  "one lined" into something like :
(curl -fsSL https://www.percona.com/get/pmm ||  wget -O - https://www.percona.com/get/pmm ) | /bin/bash  

? 
 so it executes the first or second command if the first is failed?

**Comment by shoffman-percona** [comment]
> @shoffman-percona , @nikita-b can we have this "one lined" into something like : (curl -fsSL https://www.percona.com/get/pmm || wget -O - https://www.percona.com/get/pmm ) | /bin/bash
> 
> ? so it executes the first or second command if the first is failed?

We do that now in the script but the idea is to not have big wild lines of code for average "joe user" to have to parse...

**Review by BupycHuk — APPROVED** [review]

**Review by nikita-b — APPROVED** [review]

**Comment by rnovikovP** [comment]
> average "joe user."

but now we are asking Joe to know what system he runs command .. does it Curl or Wget based  🤔

File: docs/setting-up/server/easy-install.md [hunk 1]
```diff
@@ -4,16 +4,27 @@
...
+```
+
 This script:
 
 - Installs Docker if it is not installed.
 - If a PMM Server Docker container is running, it is stopped and backed up.
 - Pulls and runs the latest PMM Server Docker image.
+- this script also has an interactive mode if you'd like to change default settings:
+```sh
+curl -fsSL https://www.percona.com/get/pmm (or wget https://www.percona.com/get/pmm)
```
Comment (denisok) [inline docs/setting-up/server/easy-install.md:24]:
```suggestion
curl -fsSLO https://www.percona.com/get/pmm (or wget https://www.percona.com/get/pmm)
```

Comment (denisok) [reply docs/setting-up/server/easy-install.md:24]: without `O` it would be console.

**Review by denisok — CHANGES_REQUESTED** [review]

**Review by catalinaadam — COMMENTED** [review]
Made a few suggestions to keep list items parallel.

File: docs/setting-up/server/easy-install.md [hunk 2]
```diff
@@ -4,16 +4,27 @@
...
 
+Using wget: 
+```sh
+wget -O - https://www.percona.com/get/pmm | /bin/bash
+```
+
 This script:
 
 - Installs Docker if it is not installed.
 - If a PMM Server Docker container is running, it is stopped and backed up.
```
[the remark this answers is not in the document]
Comment (shoffman-percona) [reply docs/setting-up/server/easy-install.md]: Updated!

File: docs/setting-up/server/easy-install.md [hunk 3]
```diff
@@ -4,16 +4,27 @@
...
 - Pulls and runs the latest PMM Server Docker image.
+- this script also has an interactive mode if you'd like to change default settings:
```
[the remark this answers is not in the document]
Comment (shoffman-percona) [reply docs/setting-up/server/easy-install.md]: Updated!

**Review by denisok — APPROVED** [review]

**Review by rasika-chivate — APPROVED** [review]

**Pull request merged**